### PR TITLE
Add chunk format check in in_forward

### PR DESCRIPTION
**Pull request by repeatedly**
This is useful for which forwarder agent sends broken chunk to aggregators.

### CODE REVIEW

File: lib/fluent/plugin/in_forward.rb [hunk 1]
```diff
@@ -125,7 +125,12 @@ def on_message(msg, chunk_size, source)
         return
       end
 
-      # TODO format error
+      # TODO: raise an exception if broken chunk is generated by recoverable situation
+      unless msg.is_a?(Array)
+        log.warn "incoming chunk is broken:", source: source, msg: msg
```
Comment (tagomoris) [inline lib/fluent/plugin/in_forward.rb]: `msg` should be `inspect` -ed for various values, ex: strings with escape code or others.

Comment (repeatedly) [inline lib/fluent/plugin/in_forward.rb]: `inspect` is done in log methods.

https://github.com/fluent/fluentd/blob/7bd529cc8007f04592c971f6b2fd9d216c60cf5d/lib/fluent/log.rb#L279

Comment (tagomoris) [inline lib/fluent/plugin/in_forward.rb]: ah, okay.

**Comment by repeatedly** [comment]
Are there another concern?
If not, I will merge the patch.

**Comment by tagomoris** [comment]
LGTM

**Pull request merged**